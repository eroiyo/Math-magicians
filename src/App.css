#root {
  width: 100vw;
  height: 100vh;
  font-family: "Montserrat", sans-serif;
  background-color: white;
  overflow-x: hidden;
}

.content {
  width: 100vw;
  height: 85.5vh;
  display: flex;
}

.calculator {
  display: flex;
  flex-direction: column;
  width: 25vw;
  height: 55vh;
  background-color: green;
  margin-left: auto;
  margin-right: auto;
  margin-top: auto;
  margin-bottom: auto;
  border-style: solid;
  border-color: grey;
}

.screen {
  overflow-y: hidden;
  overflow-x: scroll;
  display: flex;
  font-size: 50px;
  flex-direction: row-reverse;
  background-color: darkgrey;
  width: 100%;
  height: 25%;
}

.options {
  display: grid;
  background-color: grey;
  grid-gap: 2px;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
  width: 100%;
  height: 100%;
}

.option {
  display: flex;
  align-items: center;
  align-content: center;
  background-color: white;
}

.blue {
  background-color: #2495e0;
}

span {
  margin-left: auto;
  margin-right: auto;
}

.zero {
  grid-column: span 2;
}

h1 {
  margin-bottom: 4%;
}

p {
  margin-bottom: 2.5%;
}
