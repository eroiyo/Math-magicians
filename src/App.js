import './App.css';
import Calculator from './components/Calculator';
import React from 'react';

class App extends React.Component {
  constructor(props) {
    super(props);
  }

  /* eslint-enable */
  render() {
    return (
      <div className="App">
        <div className="App-body">
          <Calculator />
        </div>
      </div>
    );
  }
}

export default App;