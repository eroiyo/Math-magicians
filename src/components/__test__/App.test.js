import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../App';

describe('App', () => {
  it('the app renders correctly', () => {
    const tree = renderer.create(<App />);

    expect(tree).toMatchSnapshot();
  });
});