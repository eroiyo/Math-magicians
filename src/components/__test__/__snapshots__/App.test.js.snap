// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App the app renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="App-body"
  >
    <header>
      <nav
        className="main-nav"
      >
        <input
          id="check"
          type="checkbox"
        />
        <i
          className="fas fa-bars"
        />
        <a
          className="logo"
          href="index.html"
        >
          Math Magician
        </a>
        <ul
          className="navlinks"
        >
          <li>
            <a
              className="link"
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li>
            <a
              className="link"
              href="/calculator"
              onClick={[Function]}
            >
              Calculator
            </a>
          </li>
          <li>
            <a
              className="link"
              href="/quote"
              onClick={[Function]}
            >
              Quote
            </a>
          </li>
        </ul>
      </nav>
    </header>
    <div
      className="content"
    >
      <div
        id="welcome"
      >
        <h1>
          Welcome to our page
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Mauris ultrices eros in cursus turpis massa tincidunt. Eget sit amet tellus cras adipiscing enim. Sed arcu non odio euismod lacinia at quis risus. Suspendisse ultrices gravida dictum fusce ut placerat orci. Potenti nullam ac tortor vitae purus faucibus. Tempor nec fe ugiat nisl pretium fusce. Pulvinar etiam non quam lacus suspendisse faucibus. Eu sem i nteger vitae justo eget magna. Sed viverra ipsum nunc aliquet. Lacinia quis vel eros d onec ac odio tempor orci. Arcu cursus euismod quis viverra nibh. Lectus proin nibh nisl condimentum id venenatis. Ac tortor dignissim convallis aenean et tortor at risus viverra.
        </p>
        <p>
          Est ante in nibh mauris. Blandit turpis cursus in hac habitasse platea dictumst qu isque sagittis. Dui faucibus in ornare quam viverra. Lacus sed turpis tincidunt id a liquet risus feugiat in ante. Enim ut sem viverra aliquet eget. Ornare lectus sit am et est placerat in egestas erat. Id porta nibh venenatis cras sed felis eget. Magna etiam tempor orci eu lobortis elementum nibh tellus. Imperdiet massa tincidunt nunc pulvinar sapien. Natoque penatibus et magnis dis parturient montes nascetur ridiculu s. Dignissim suspendisse in est ante in nibh mauris cursus mattis. Morbi tristique s enectus et netus et malesuada fames ac turpis. Pharetra pharetra massa massa ultrici es mi. Volutpat odio facilisis mauris sit amet massa vitae tortor condimentum. Fauci bus purus in massa tempor. Libero volutpat sed cras ornare arcu dui.
        </p>
      </div>
    </div>
  </div>
</div>
`;
